index.ts

```diff
@@ -16,6 +16,14 @@
 
 'use strict';
 
+declare global {
+  namespace NodeJS {
+    export interface Global {
+      _google_trace_agent: any;
```

Comment:
Will this type be updated to an "agent"?

Reply:
Yes, in the next part.